### ISSUE TITLE
[Bug] Windows Error Code: 5

### ISSUE THREAD

**Issue by happyTonakai** [pit-ray/win-vind#122]
# About Bug
When win-vind starts up with Windows, the following error shows

![issue](https://user-images.githubusercontent.com/21305646/190062836-77dd224e-67e2-47ef-ac19-873311384240.png)

A part of log is as follows
```
========== System Infomation ==========
[Windows]
      Edition: Windows 10 Pro
      Version: 21H1
Build Numbers: 10.0.19043
 Architecture: x64

[win-vind]
      Version: 4.3.3
=======================================

[Error] open_startmenu failed. An runtime exception occurred from void __cdecl vind::bind::JumpToActiveWindow::sprocess(void). GetForegoundWindow return nullptr (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. An runtime exception occurred from void __cdecl vind::util::move_cursor(int,int). cannot send the moving event of the mouse (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
[Error] move_cursor_right failed. Cannot refresh all key state. An runtime exception occurred from void __cdecl vind::core::InputGate::release_keystate<const class vind::core::KeyCode*>(const class vind::core::KeyCode *,const class vind::core::KeyCode *). failed sending keyboard event (void __cdecl vind::bind::BindedFunc::error_process(const class std::exception &) const)
[Error] Access is denied.
```

# Steps to reproduce.
1. Select Start up with Windows.
2. Reboot system.
3. Don't enter the password immediately, wait for one or two minutes. Win10 will start all startup items even if I don't enter the password to login yet.
4. Wait for screen to sleep (this step may be optional, but I can 100% reproduce the bug if I wait for the screen to turn black).
5. Wake up the system to login and the error message shows up.

# Environments

|Environments|Delete the example and fill in your information.|
|:---:|:---|
|Windows Edition|10 Pro|
|Windows Version|21H1|
|Windows Build|19043.1949|
|Architecture|64bit|
|win-vind Version|4.3.3|
|win-vind Install Type|winget|

**Comment by pit-ray** [comment]
I try to reproduce it.
By the way, does this issue occur in the default .vindrc?
Thanks.

**Comment by happyTonakai** [comment]
> I try to reproduce it. By the way, does this issue occur in the default .vindrc? Thanks.

Yes, this issue also occurs with a blank `.vindrc`.

**Comment by pit-ray** [comment]
I tried on three environment.

- Windows 10 Home
I could not reproduce it.

- Windows 11 Home
I could not reproduce it.

- Windows 10 Pro
I could reproduce it.

Therefore, I guess it due to edition difference. I continue the investigation.

**Comment by happyTonakai** [comment]
Well I don't think it should be closed since it's not fixed yet.

**Comment by pit-ray** [comment]
I added tthis issue into https://github.com/users/pit-ray/projects/2 with high priority.

**Comment by pit-ray** [comment]
This issue was already resolved in cf4b80895a76a7244c93175def00dc1ccdd1d88d , so I will close it.
